backend/scraper/scrape_heb.py

```diff
@@ -1 +1,100 @@
-# TODO scrape items from HEB
+from bs4 import BeautifulSoup
+import requests
+
+# The goal is to have a get_products functions that returns a list of prices given a string query,
+# Parse_link function that gets the name, price,
+# price per unit, and image URL of a specific item. See Ryan's Walmart scraper for examples.
+
+# HEB Scraper Class
+class HEBScraper():
+    def __init__(self):
+        pass
+
+    def getProudcts(self, product):
+        # Get the links of all items of this type
+        product_links = []
+        self.getLinks(product, product_links)
+
+        # Use the links to parse and obtain information
+        item_details = []

```

Comment:
This could me simplified with a [Python list comprehension](https://www.pythonforbeginners.com/basics/list-comprehensions-in-python/)